serverlessworkflow/modules/ROOT/pages/data-index/common/_dataindex_deployment_operator.adoc

```diff
@@ -0,0 +1,218 @@
+
+You can create a folder called infra/dataindex that would contain the following four files:
+
+* kustomization.yaml
+* 01-postgres.yaml
+* 02-dataindex.yaml
+* application.properties
+
```

Comment:
What about the `platform/` , `workflows` folders and the `service_discovery` are they needed? I notice them in output in commands, maybe an oversight? Not sure from my side, please confirm.